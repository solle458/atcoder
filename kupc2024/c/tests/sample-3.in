11
.c..c.s..s.
.c.c..s..s.
.cc...s..s.
.c.c..s..s.
.c..c..ss..
...........
.sss...ccc.
.s..s.c....
.sss..c....
.s....c....
.s.....ccc.
