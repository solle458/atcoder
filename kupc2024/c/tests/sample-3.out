Yes
11.00000000 10.60555128
0.39444872 0.00000000
